docker/service/src/main/java/uk/nhs/adaptors/scr/fhirmappings/ObservationMapper.java

```diff
@@ -0,0 +1,104 @@
+package uk.nhs.adaptors.scr.fhirmappings;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Observation;
+import org.hl7.fhir.r4.model.Period;
+
+import uk.nhs.adaptors.scr.exceptions.FhirMappingException;
+import uk.nhs.adaptors.scr.models.GpSummary;
+import uk.nhs.adaptors.scr.models.gpsummarymodels.EndTime;
+import uk.nhs.adaptors.scr.models.gpsummarymodels.ObservationObject;
+import uk.nhs.adaptors.scr.utils.DateUtil;
+
+public class ObservationMapper {
+
+    public static void mapObservations(GpSummary gpSummary, List<Observation> observations) throws FhirMappingException {
+        List<ObservationObject> observationList = new ArrayList<>();
```

Comment:
Can be java stream

Reply:
Should this and also condition mapper be non static, I can make these streams however it needs to be non static. What is the best approach? For consistency the other mapper before are static

Reply:
Doesn' have to be non-static